package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "npx prettier --write . && next dev",
```

Comment:
Hi @batteries76 ,

call me paranoid, but I prefer not to have any linter fix any linting issues automatically, **unless** I've had a chance to review them :sweat_smile: 
I prefer that `npm run dev` just spins up my local dev environment, tbh.
Can we have a separate run script for `npx prettier --write` instead? :pray:

Reply:
Yeah sure, can do. I think on balance I prefer that too.
I guess I was thinking that it might lead to a lot of "please can you resubmit your PR after running Prettier" back and forward. 
Happy to go with your suggestion.